### DISCUSSION TITLE
I want to be able to work for EKS and AKS with the argument of starboard kube-bench command.

### DISCUSSION THREAD

**Discussion by hiropinponpan** [aquasecurity/starboard#107] [Development]
kube-bench(https://github.com/aquasecurity/kube-bench) supports cases such as EKS and AKS where the master node cannot be checked, but starboard(https://github.com/ aquasecurity/starboard) does not support this, 
so please support it with an argument.
I had confirmed that it works by rewriting the source code (https://github.com/aquasecurity/starboard/blob/master/pkg/kubebench/scanner.go) and not checking the mastar node.

**Comment by lizrice** [comment]
For EKS and GKE there are now dedicated versions of the CIS benchmark (eks-1.0 and gke-1.0 kube-bench test configs) and it would be better to use those on the appropriate platforms. I just opened an [enhancement issue about auto-detecting this in kube-bench](https://github.com/aquasecurity/kube-bench/issues/661).
